### PR TITLE
Fix Ingress callouts, links, nav, and templating

### PR DESCRIPTION
**Pull request by cody-clark**
Small updates to previous Ingress doc rework

### CODE REVIEW

**Comment by cody-clark** [comment]
[Minikube Ingress Task](https://deploy-preview-12720--kubernetes-io-master-staging.netlify.com/docs/tasks/access-application-cluster/ingress-minikube/) 
* Fixed callout formatting and broken numbered lists

[Ingress Controllers](https://deploy-preview-12720--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/ingress-controllers/) 
* Removed toc_hide: true 
* Added overview tag to fit template standards 
* Added two newly added ingress controllers

[Ingress](https://deploy-preview-12720--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/ingress/) 
* Deleted Ingress controller section

**Comment by cody-clark** [comment]
PTAL Steve. This PR fixes some weird formatting that the shortcodes seem to introduce along with some other small fixes.

/assign @steveperry-53

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 1]
```diff
@@ -187,28 +181,27 @@ The following file is an Ingress resource that sends traffic to your Service via
 
 1. Add the following line to the bottom of the `/etc/hosts` file. 
 
-   ```shell
-   172.17.0.15 hello-world.info
-   ```
+    ```shell
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/ingress-minikube.md]: This isn't a shell command (was also wrong in the original).

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -2,23 +2,30 @@
...
-
-## Ingress controllers
+{{% capture overview %}}
 
 In order for the Ingress resource to work, the cluster must have an ingress controller running. 
-This is unlike other types of controllers, which run as part of the `kube-controller-manager` binary, 
-and are typically started automatically with a cluster. Choose the ingress controller implementation 
+
+Unlike other types of controllers which run as part of the `kube-controller-manager` binary, Ingress controllers 
+and are not started automatically with a cluster. Choose the ingress controller implementation
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/ingress-controllers.md]: s/and are/are

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md (hunk 1 above)
Comment (cody-clark) [reply content/en/docs/tasks/access-application-cluster/ingress-minikube.md]: Removed syntax according to https://github.com/kubernetes/website/pull/12783#issuecomment-467223409

File: content/en/docs/concepts/services-networking/ingress.md [hunk 3]
```diff
@@ -57,48 +57,7 @@ of this controller if you are using GCE/GKE.
...
-* [Traefik](https://github.com/containous/traefik) is a fully featured ingress controller
-  ([Let's Encrypt](https://letsencrypt.org), secrets, http2, websocket), and it also comes with commercial
-  support by [Containous](https://containo.us/services).
-
-You may deploy [any number of ingress controllers](https://git.k8s.io/ingress-nginx/docs/user-guide/multiple-ingress.md#multiple-ingress-controllers) within a cluster.
-When you create an ingress, you should annotate each ingress with the appropriate
-[`ingress.class`](https://git.k8s.io/ingress-gce/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster) to indicate which ingress
-controller should be used if more than one exists within your cluster.
-If you do not define a class, your cloud provider may use a default ingress provider.
+[ingress controller](/docs/concepts/services-networking/ingress-controllers) you may choose from.
```
Comment (steveperry-53) [inline content/en/docs/concepts/services-networking/ingress.md]: Plural? a number of ingress controllers

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 4]
```diff
@@ -118,10 +116,7 @@ This page shows you how to set up a simple Ingress which routes requests to Serv
     http://172.17.0.15:31637
     ```
     
-    {{< note >}}
-    Katacoda environment only: at the top of the terminal panel, click the plus sign, and then click 
-    **Select port to view on Host 1**. Enter the NodePort, in this case `31637`, and then click **Display Port**
-    {{< /note >}}
+    {{< note >}}Katacoda environment only: at the top of the terminal panel, click the plus sign, and then click **Select port to view on Host 1**. Enter the NodePort, in this case `31637`, and then click **Display Port**{{< /note >}}
```
Comment (steveperry-53) [inline content/en/docs/tasks/access-application-cluster/ingress-minikube.md]: Period at end of this sentence.

**Pull request merged**